docs/basics/support.md

```diff
@@ -0,0 +1,24 @@
+# DockSTARTer Videos, Tutorials, Guides, and Blogs
```

Comment:
With the new changes to the navigation format (mentioned in my other comment) the title of the page (first heading) will be the title used for the navigation link. This is pretty long for that purpose. Can we just call it Support? If so we might need to rename the heading below this.

Reply:
support sounds good to me